packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -42,11 +40,13 @@ export class BLSCryptographyClient {
    */
   public async combinePartialBlindedSignatures(
     blindedMessage: string,
+    pubKey: string,
+    polynomial: string,
+    threshold: number,
```

Comment:
Ah cool I wasn't aware of that. Do you know how the decoding works?  Unless it's straightforward, would you be comfortable saving this as a TODO comment?

Reply:
I think it might be available in the Rust SDK, but looking at the JS interface now, it does not appear to be exposed if it is. Worst case scenario here is that the verification fails if these variables are mixed up, so its not a security concern. No need to do anything, LGTM.